src/main/resources/locales/cy/birmingham-administrative-court-daily-cause-list.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Rhestr Achosion Dyddiol Llys Gweinyddol Birmingham",
+  "heading": "Rhestr Achosion Dyddiol Llys Gweinyddol Birmingham",
+  "importantInformationText1": "Mae gwrandawiadau'n cael eu cynnal yn gyhoeddus oni nodir yn wahanol. Wrth ystyried y defnydd o dechnoleg ffôn a fideo, bydd y farnwriaeth yn ystyried egwyddorion cyfiawnder agored. Gall y llys eithrio arsylwyr lle bo angen sicrhau gweinyddiaeth briodol cyfiawnder.",
+  "importantInformationHeading2": "Dyfarniadau",
+  "importantInformationText2": "Bydd dyfarniadau a draddodir gan y barnwr o bell yn cael eu rhyddhau trwy gylchrediad i gynrychiolwyr y partïon trwy e-bost a'u rhyddhau i'r Archifau Cenedlaethol. Ystyrir na fydd y dyddiad a'r amser ar gyfer traddodi yn digwydd cyn yr amser a restrwyd. Gellir darparu copi o'r dyfarniad ar ffurf derfynol fel y'i traddodwyd ar ôl yr amser hwnnw, ar gais drwy e-bost.",
+  "tableHeaders": [
+    "Lleoliad",
+    "Barnwyr",
+    "Amser",
+    "Rhif yr Achos",
```

Comment:
Use lower case for Achos, to be consistent with other non-strategic lists

Reply:
fixed.